qr-code/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	font-size: 15px;
+	font-family: 'Outfit', sans-serif;
+    background-color: hsl(212, 45%, 89%);
+}
+.box {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	height: 450px;
+	width: 280px;
+	background-color: white;
+	border-radius: 15px;
+    
+}
+
+.text {
+	margin-top: 5px;
+	padding: 10px 10px;
+	text-align: center;
+}
+.text-large {
+    margin-top: 20px;
+	font-size: 20px;
+	font-weight: bold;
+}
+
+.text-small {
+	margin-top: 15px;
+    color: hsl(220, 15%, 55%)
+}
+
+img {
+    width: 98%;
```

Comment:
raczej niespotykany sposób nadawania szerokości obrazkom :) Raczej 100%, albo wartość pixelowa na twardo, a jeżeli ma być responsywne, no to media-queries.